dali/pipeline/data/tensor_vector.cc

```diff
@@ -1176,13 +1122,13 @@ bool TensorVector<Backend>::shares_data() const {
 
 template class DLL_PUBLIC TensorVector<CPUBackend>;
 template class DLL_PUBLIC TensorVector<GPUBackend>;
-template void TensorVector<CPUBackend>::Copy<CPUBackend>(const TensorVector<CPUBackend>&, AccessOrder, bool);  // NOLINT
-template void TensorVector<CPUBackend>::Copy<GPUBackend>(const TensorVector<GPUBackend>&, AccessOrder, bool);  // NOLINT
-template void TensorVector<GPUBackend>::Copy<CPUBackend>(const TensorVector<CPUBackend>&, AccessOrder, bool);  // NOLINT
-template void TensorVector<GPUBackend>::Copy<GPUBackend>(const TensorVector<GPUBackend>&, AccessOrder, bool);  // NOLINT
-template void TensorVector<CPUBackend>::Copy<CPUBackend>(const TensorList<CPUBackend>&, AccessOrder);  // NOLINT
-template void TensorVector<CPUBackend>::Copy<GPUBackend>(const TensorList<GPUBackend>&, AccessOrder);  // NOLINT
-template void TensorVector<GPUBackend>::Copy<CPUBackend>(const TensorList<CPUBackend>&, AccessOrder);  // NOLINT
-template void TensorVector<GPUBackend>::Copy<GPUBackend>(const TensorList<GPUBackend>&, AccessOrder);  // NOLINT
+template void TensorVector<CPUBackend>::Copy<CPUBackend>(const TensorVector<CPUBackend> &,
+                                                         AccessOrder, bool);
+template void TensorVector<CPUBackend>::Copy<GPUBackend>(const TensorVector<GPUBackend> &,
+                                                         AccessOrder, bool);
+template void TensorVector<GPUBackend>::Copy<CPUBackend>(const TensorVector<CPUBackend> &,
+                                                         AccessOrder, bool);
+template void TensorVector<GPUBackend>::Copy<GPUBackend>(const TensorVector<GPUBackend> &,
+                                                         AccessOrder, bool);
```

Comment:
How about putting every instantiation in a separate line and adding NOLINT? Your call

Reply:
I removed it as I felt the NOLINT was a bit of overkill here. It's aligned quite well.